raft/rawnode.go

```diff
@@ -123,6 +123,14 @@ func (rn *RawNode) Tick() {
 	rn.raft.tick()
 }
 
+// TickQuiesced advances the internal logical clock by a single tick without
+// performing any other state machine processing. It allows the caller to avoid
+// periodic heartbeats and elections when all of the peers in a Raft group are
+// known to be at the same state.
```

Comment:
Point out that one would call exactly one of `Tick()` or `TickQuiesced()` periodically, depending on (duh) whether the group is "active" or "quiesced".